### DISCUSSION TITLE
Wrong visualization copilot chat - Visual Studio Community 2022

### DISCUSSION THREAD

**Discussion by fqh4** [community/community#60919] [Copilot Conversations]
### Select Topic Area

Bug

### Body

i have reinstalled the VS, the extension but none of these fix the issue...

Microsoft Visual Studio Community 2022 (64-bit) - Current
Version 17.6.5

![image](https://github.com/community/community/assets/125499340/b466b139-b7f6-4d7e-8653-586e0f83ab34)

![image](https://github.com/community/community/assets/125499340/660853c5-a80b-4428-9502-54aaaa2f428a)

**Comment by samus-aran** [comment]
Hi @fqh4 👋 

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, [including posting the same comment in multiple places](https://github.com/orgs/community/discussions/62042). As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
